lib/kv-redis.js

```diff
@@ -97,14 +97,18 @@ RedisKeyValueConnector.prototype.ping = function(cb) {
  *
  * @param {String} command The Redis command to execute.
  * @param {Array} args List of options for the given command.
- * @callback {Function} callback
+ * @options {Object} opts (optional) This is currently not used but included
+ * to match the signature in the loopback-datasource-juggler
+ * @callback {Function} cb Callback invoked on redis command
  * @param {Error} err Error object.
  * @param {*} result The result of the command execution.
  *
- * @header RedisKeyValueConnector.prototype.execute(command, args, cb)
+ * @header RedisKeyValueConnector.prototype.execute(command, args, opts, cb)
  */
-RedisKeyValueConnector.prototype.execute = function(command, args, cb) {
-  if (cb === undefined && typeof args === 'function') {
+RedisKeyValueConnector.prototype.execute = function(command, args, opts, cb) {
```

Comment:
Please use `options` for consistence with existing code. Don't forget to update the jsdoc comment too!

```suggestion
RedisKeyValueConnector.prototype.execute = function(command, args, options, cb) {
```